src/modules/admin/components/users/UserAuditPage.tsx

```diff
@@ -1,25 +1,134 @@
+import useDebouncedState from '@/hooks/useDebouncedState';
 import useSafeParams from '@/hooks/useSafeParams';
+import ClientAccessSummaryTable from '@/modules/admin/components/users/ClientAccessSummaryTable';
+import EnrollmentAccessSummaryTable from '@/modules/admin/components/users/EnrollmentAccessSummaryTable';
 import { useUser } from '@/modules/dataFetching/hooks/useUser';
+import CommonSearchInput from '@/modules/search/components/CommonSearchInput';
+import { AdminDashboardRoutes } from '@/routes/routes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+type EntityType = 'clients' | 'enrollments';
 
-const UserAuditPage: React.FC = () => {
+const toggleItems: ToggleItem<EntityType>[] = [
+  {
+    value: 'clients',
+    label: 'Clients',
+    Icon: ClientIcon,
+  },
+  {
+    value: 'enrollments',
+    label: 'Enrollments',
+    Icon: EnrollmentIcon,
+  },
+];
+
+interface Props {
+  entityType: EntityType;
+}
+const UserAuditPage: React.FC<Props> = ({ entityType }) => {
   const { userId } = useSafeParams() as { userId: string };
   const { user, loading } = useUser(userId);
+  const [search, setSearch, debouncedSearch] = useDebouncedState<string>('');
+  const navigate = useNavigate();
+  //const defaultStartDate = useMemo<string>(() => {
+  //  const today = new Date();
+  //  const twoWeeksAgo = subWeeks(today, 2);
+  //  return formatISO(twoWeeksAgo);
+  //}, []);
 
   if (!user && loading) return <Loading />;
   if (!user) return <NotFound />;
 
-  // TODO: support toggling between Access History and Edit History
+  const handleTabChange = (value: EntityType) => {
+    switch (value) {
+      case 'clients':
+        navigate(
+          generateSafePath(AdminDashboardRoutes.USER_CLIENT_ACCESS_HISTORY, {
+            userId,
+          })
+        );
+        break;
+      case 'enrollments':
+        navigate(
+          generateSafePath(
+            AdminDashboardRoutes.USER_ENROLLMENT_ACCESS_HISTORY,
+            { userId }
+          )
+        );
+        break;
+    }
+  };
+
   return (
     <>
       <PageTitle title={`Access History for ${user.name}`} />
       <Paper>
-        <UserAccessHistoryTable userId={userId} />
+        <Box mt={2} px={2}>
+          <Typography variant='subtitle1' sx={{ mb: 2 }}>
+            {`Data on this page may be delayed by up to an hour. Recent user activity may not be immediately visible.`}
+          </Typography>
+          <Typography sx={{ fontWeight: 600, mb: 0.5 }}>
+            View user access by
+          </Typography>
+          <CommonToggle
+            sx={{ mb: 3 }}
+            value={entityType}
+            onChange={handleTabChange}
+            items={toggleItems}
+            size='small'
+            variant='gray'
+          />
+          {entityType == 'clients' && (
+            <CommonSearchInput
+              label='Search client access'
+              name='searchClients'
+              placeholder='Search by client name/ID'
```

Comment:
We can export and use this placeholder, since it uses the same underlying client search

https://github.com/greenriver/hmis-frontend/blob/78e2b44689bca8e5312770419419f91c1c8a0726/src/modules/search/components/ClientTextSearchInput.tsx#L17-L18

Reply:
Or, you can use `ClientTextSearchInput` component directly if possible.

Reply:
nice, yes I'll drop in ClientTextSearchInput.